nursery/persist-via-print-processors-registry-key.yml

```diff
@@ -0,0 +1,25 @@
+rule:
+  meta:
+    name: persist via Print Processors registry key
+    namespace: persistence
+    authors:
+      - j.j.vannielen@utwente.nl
+    scopes:
+      static: function
+      dynamic: call
+    att&ck:
+      - Persistence::Boot or Logon Autostart Execution::Print Processors [T1547.012]
+    references:
+      - https://stmxcsr.com/persistence/print-processor.html
+  features:
+    - or:
+      - and:
+        - match: set registry value
+        - string: /SYSTEM\\(CurrentControlSet|ControlSet001)\\Control\\Print\\Environments\\.*\\Print Processors\\/i
+        - string: /^Driver$/i
+      - and:
+        - or:
+          - match: copy file
+          - match: move file
+          - match: host-interaction/file-system/write
+        - string: /\\spool\\PRTPROCS\\/i
```

Comment:
with the `dynamic: call` scope this would only work for static processing, is that intended?

Reply:
It is intended to work for dynamic as well.
When https://github.com/mandiant/capa-rules/pull/952 is merged, this should work for dynamic as well (as it changes the scope of `copy file`, `move file`, and `write file to the `call` scope).
Or is there another reason why it wouldn't work for dynamic analysis?

Reply:
nope, you're right, thanks!